docs/satellite-repos.md

```diff
@@ -0,0 +1,20 @@
+# Satellite repos
+
+
+Satellite repositories provide functionality or site content that is optionally included as part of the website, but not included in this repository. Some satellite repos exist to allow certain functionality to be run or tested without needing to set up the full website.
+
+!!! note "Thinking about making a new satellite repo?"
```

Comment:
FYI: since this PR was opened, @Scotchester created a new internal page (see "Setting up new project code for consumerfinance.gov" in the platform wiki) that goes into more detail about when to use satellite apps. This content was moved from [this](https://cfpb.github.io/cfgov-refresh/new-projects/) existing doc page which was tagged to be moved internal as part of our IA review.

Given all that, I think this note is still okay, but wanted to ping @Scotchester in case he feels differently.